mappings/net/minecraft/util/dynamic/GlobalPos.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4208 net/minecraft/util/GlobalPos
+CLASS net/minecraft/class_4208 net/minecraft/util/dynamic/GlobalPos
```

Comment:
Why is `GlobalPos` going into the dynamic package? Does it work with dynamics?

Reply:
Yes, this is like dynamic serializable uuid (used by villager brains) and does not implement `Position` or `WorldPosition`

Reply:
Maybe a different class name is in order, then. Because `GlobalPos` tells me that it works with `BlockPos` and nothing else.